src/workloads/basic/ReadOnlyMultiThreaded.yml

```diff
@@ -0,0 +1,91 @@
+SchemaVersion: 2018-07-01
+Owner: Product Performance
+Description: |
+  Runs a multi-threaded, read-only workload comprising of `findOne` commands that all target the 
+  same document. The workload inserts a single document and then spawns multiple worker threads to 
+  retrieve it.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+  Database: &Database test
+  Collection: &Collection test
+  PaddingLength: &PaddingLength 64 # bytes
+  WarmUpPeriod: &WarmUpPeriod 30 seconds
+  RunPeriod: &RunPeriod 5 minutes
+  Threads: &Threads 64
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 500
+
+Actors:
+- Name: CRUDOps
+  Type: CrudActor
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *Database
+    Collection: *Collection
+    Operations:
+    - OperationName: drop
+  - Repeat: 1 # only 1 document is needed for this workload
+    Database: *Database
+    Collection: *Collection
+    Operations:
+    - OperationName: insertOne
+      OperationCommand:
+        Document:
+          _id: 1
+          padding: {^FastRandomString: {length: *PaddingLength}}
+  - *Nop
+  - *Nop
+- Name: RunCommand
+  Type: RunCommand
+  Threads: *Threads
+  Phases:
```

Comment:
You could also consider using [OnlyActiveInPhases](https://github.com/mongodb/genny/blob/c4ce0d6dbe195629a3348eba9573f25381890683/docs/using.md#orgf9c328f) but I'm not convinced it would save you anything here.